spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewritePositionDeleteFilesAction.java

```diff
@@ -136,7 +136,7 @@ public void testEmptyTable() {
   @Test
   public void testUnpartitioned() throws Exception {
     Table table = createTableUnpartitioned(2, SCALE);
-    List<DataFile> dataFiles = dataFiles(table);
+    List<DataFile> dataFiles = TestHelpers.dataFiles(table, true);
```

Comment:
Seeing all these test pass in true, I'm thinking it may be ok to just always include column stats , then, to avoid passing true all the time. 

The not includeColumnStats is to increase performance, but I think it doesnt matter that much for tests.  I dont think any test logic would break if we have column stats.  

(Sorry I think it was probably my original comment to pass in false in the Partitions Table tests, as at the time we had an easier choice to include it or not)

Reply:
@szehon-ho Sounds good, let me revert it back.